Jenkinsfile

```diff
@@ -1,5 +1,6 @@
 buildPlugin(
     platforms: ['linux'],
+    jenkinsVersions: [null],
```

Comment:
Why? That is the default.

Reply:
It is a part of my fiasco with Jenkinsfiling tests against 2.102. Maybe we need to update the blogpost, it does not really work in several plugins due to Maven Enforcer. The blogpost should suggest a flag for disabling it